ebean-test/src/test/java/org/tests/query/sqlquery/SqlQueryTests.java

```diff
@@ -224,6 +241,44 @@ public CustDto map(ResultSet rset, int rowNum) throws SQLException {
 
   private static final CustMapper CUST_MAPPER = new CustMapper();
 
+  @Test
+  void queryUsingTransaction() {
+    ResetBasicData.reset();
+
+    try (Transaction txn = DB.createTransaction()) {
+      String sql = "select id, name, status from o_customer where name is not null";
+      AtomicInteger counter = new AtomicInteger();
+      DB.sqlQuery(sql)
+        .usingTransaction(txn)
```

Comment:
The test would/should pass also without this line.
I'm wondering if the test should check if the passed transaction is REALLY used.
Maybe check, if `txn.logPrefix()` appears in logged sqls?

Reply:
I'll adjust this in master.  Having merged the jakarta.xml change it's now hard to work on this branch for some reason.